sqlalchemy_redshift/dialect.py

```
@@ -1333,6 +1524,11 @@ def dbapi(cls):
                 'redshift_connector to use this sqlalchemy dialect.'
             )
 
+    @classmethod
+    def dbapi(cls):
```

Ruff (0.14.0)

Missing return type annotation for classmethod `dbapi`

(ANN206)